velox/expression/CastExpr-inl.h

```diff
@@ -126,6 +126,23 @@ StringView convertToStringView(
   return StringView(startPosition, writePosition - startPosition);
 }
 
+inline bool hasWhitespace(const StringView& value) {
+  if (value.size() > 0) {
```

Comment:
nit: perhaps,

```
if (value.size() == 0) {
   return false;
}
```

Why is the code currently return true when value is an empty string?